### ISSUE TITLE
Remove course id from build overview page

### ISSUE THREAD

**Issue by MaximilianAnzinger** [ls1intum/Artemis#9800]
### Is your feature request related to a problem?

All the tables in `Course Management > Build Overview` (Running build jobs, Queued build jobs, and Finished build jobs) feature the course id. This is irrelevant information to users since in this view they only see jobs related to this course.

### Describe the solution you'd like

Remove course id from tables to simplify UI.

**Comment by krusche** [comment]
I agree, the table is currently reused from the admin build overview.

On the admin page, the course id makes sense. In the course build overview, we should hide this column 👍🏻

**Comment by hanna20022005** [comment]
Should be fixed with PR #11612 :>
